Source/WebCore/page/IntelligenceTextEffectsSupport.cpp

```diff
@@ -135,5 +135,15 @@ void decorateWritingToolsTextReplacements(Document& document, const SimpleRange&
 }
 #endif
 
+void setSelection(Document& document, const SimpleRange& scope, const CharacterRange& range)
```

Comment:
Not sure why this needs to be a method? It's called in one place, the logic is quite simple, and not directly related to intelligence effects.

Reply:
ya, I mainly just did it for consistency and to avoid having extra logic in WebPage. I’ll remove the method and move the logic.